src/cost_functions.jl

```diff
@@ -0,0 +1,21 @@
+"""
+Returns the amount of misclassified examples, i.e. how many tuples with non-matching entries are there in `results`.
+# Arguments
+- `results::AbstractVector{Tuple{Int64, Int64}}`: the vector of tuples, where each tuple is in the form `Tuple{expected_output, actual_output}`.
+"""
+function accuracy(results::AbstractVector{Tuple{Int64,Int64}})
```

Comment:
small thing: if this is calculating misclassifications, then let's call is `misclassification` rather than `accuracy`. Accuracy means the opposite: higher the number the better.

Reply:
Yes, you are right! We confused the terminology.